### PR TITLE
ni-utils: Move wireless udhcpd configuration file from os-common

### PR DESCRIPTION
**Pull request by gratian**
Signed-off-by: Gratian Crisan <gratian.crisan@ni.com>

### CODE REVIEW

**Comment by gratian** [comment]
Attn: @ni/rtos

File: recipes-ni/ni-utils/files/udhcpd.wlan0.conf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+start 172.16.0.10
```
Comment (shruthi-ravi) [inline recipes-ni/ni-utils/files/udhcpd.wlan0.conf:1]: Does it make sense to add this file to the busybox recipe in a udhcpd package instead, like the ifplugd.conf file [here](https://github.com/ni/meta-nilrt/blob/nilrt/master/dunfell/recipes-core/busybox/files/ifplugd.conf)?

Comment (gratian) [reply recipes-ni/ni-utils/files/udhcpd.wlan0.conf:1]: Yeah, I'll move it. I really don't like tangling more things into busybox - we've been slowly replacing busybox utilities with the full functionality stand alone ones but I guess for now that makes more sense.

Comment (gratian) [reply recipes-ni/ni-utils/files/udhcpd.wlan0.conf:1]: Closing this PR in favor of a new one against the busybox recipe.

**Pull request closed without merging**